static/js/income_chart_options.js

```diff
@@ -0,0 +1,132 @@
+      type: "column",
+      height: `${height}px"`,
+      pointInterval: 24 * 3600 * 1000,
+      events: {
+        load: function () {
+          $("#load-income-btn").removeClass("d-none")
+        },
+      },
+    },
+    title: {
+      text: title,
+    },
+    credits: {
+      enabled: false,
+    },
+    legend: {
+      enabled: true,
+    },
+    plotOptions: {
+      column: {
+        stacking: "stacked",
+        dataLabels: {
+          enabled: false,
+        },
+        pointInterval: 24 * 3600 * 1000,
+      },
+      series: {
+        turboThreshold: 10000,
+      },
+    },
+    xAxis: {
+      type: "datetime",
+      range: 31 * 24 * 60 * 60 * 1000,
+      labels: {
+        formatter: function () {
+          var epoch = timeToEpoch(this.value)
+          var orig = this.axis.defaultLabelFormatter.call(this)
+          return `${orig}<br/>Epoch ${epoch}`
+        },
+      },
+    },
+    yAxis: [
+      {
+        title: {
+          text: `Income [${currency}]`,
+        },
+        opposite: false,
+        labels: {
+          formatter: function () {
+            return currency === "ETH" ? trimToken(this.value) : trimCurrency(this.value)
+          },
+        },
+      },
+    ],
+    series: [
+      {
+        name: "Execution Income",
+        data: executionIncomeHistory,
+        showInNavigator: false,
+        dataGrouping: {
+          enabled: false,
+        },
+      },
+      {
+        name: "Consensus Income",
+        data: incomeHistory,
+        showInNavigator: true,
+        dataGrouping: {
+          enabled: false,
+        },
+      },
+    ],
+    tooltip: {
+      split: false,
+      shared: true,
+      formatter: (tooltip) => {
+        var text = ``
+        var total = 0
+
+        // time range for hovered point
+        const ts = tooltip.chart.hoverPoints[0].x
+        const startEpoch = timeToEpoch(ts)
+        const timeForOneDay = 24 * 60 * 60 * 1000
+        const endEpoch = timeToEpoch(ts + timeForOneDay) - 1
+        const startDate = luxon.DateTime.fromMillis(ts)
+        const endDate = luxon.DateTime.fromMillis(epochToTime(endEpoch + 1))
+        text += `${startDate.toFormat("MMM-dd-yyyy HH:mm:ss")} - ${endDate.toFormat("MMM-dd-yyyy HH:mm:ss")}<br> Epochs ${startEpoch} - ${endEpoch}<br/>`
+
+        // income
+        for (var i = 0; i < tooltip.chart.hoverPoints.length; i++) {
+          const value = tooltip.chart.hoverPoints[i].y
+          text += `<span style="color:${tooltip.chart.hoverPoints[i].series.color}">\u25CF</span>  <b>${tooltip.chart.hoverPoints[i].series.name}:</b> ${getIncomeChartValueString(value, currency)}<br/>`
+          total += value
+        }
+
+        // add total if hovered point contains rewards for both EL and CL
+        if (tooltip.chart.hoverPoints.length > 1) {
+          text += `<b>Total:</b> ${getIncomeChartValueString(total, currency)}`
+        }
+
+        return text
+      },
+      exporting: {
+        scale: 1,
+      },
```

Comment:
I think this is not needed, `exporting` is neither included nor defined on tooltips

Reply:
used wrong indentation, fixed

Reply:
:+1: But is exporting of the charts even desired? As neither of boths graphs can be exported currently. If yes, that's not working because the setting seems to get overwritten in `highcharts-global-options.js`

Reply:
I don't know tbh, the chart on the dashboard had that option set and I didn't want to remove it. I can remove it if you want to.